File: resolucoes/python/lista01/ex017.py
# Escreva um programa que pergunte a velocidade de um carro. Caso ultrapasse 80Km/h, exiba uma mensagem dizendo que o usuário foi multado. Nesse caso, exiba o valor da multa, cobrando R$5 por cada Km acima da velocidade permitida.

velocidade = float(input('Digite a velocidade do carro: '))

if velocidade > 80:
    multa = (velocidade - 80) * 5
    print(f'Você foi multado por ultrapassar 80km/h, sua multa é de: R${multa:.2f}.')
else:
    print('Você está dentro do limite de velocidade estabelecido.')

File: resolucoes/TypeScript/lista01/ex017.ts
import prompt from "prompt-sync"

export function main() {
    const p = prompt()
    const velocidade = Number(p("Qual a velocidade do carro?"))
    const cobrado = 5
    
    if (velocidade > 80) {
        // Velocidade maior que 80KM
        console.log("Velocidade superior a 80KM você será multado!")
        const total = (velocidade - 80) * 5
        console.log(`Sua velocidade foi de ${velocidade}, você será multado em R$:${total} `)
    } else {
        console.log("Velocidade inferior ou igual a 80KM, tudo ok!")
    }
}